test/base.js

```diff
@@ -9,87 +9,85 @@ var chai = require('chai'),
 
 mongoose.Promise = global.Promise;
 
-describe('Basic => ', function() {
+describe('Basic => ', function () {
 
     before(function connection(done) {
         mongoose.connection.on('open', done);
         mongoose.connection.on('error', done);
-        mongoose.connect('mongodb://127.0.0.1/mongoose-sequence-testing', {useMongoClient: true});
+        mongoose.connect('mongodb://127.0.0.1/mongoose-sequence-testing', { useMongoClient: true });
     });
 
     after(function destroyingAll(done) {
-        mongoose.connection.db.dropDatabase(function(err) {
+        mongoose.connection.db.dropDatabase(function (err) {
             if (err) return done(err);
             mongoose.disconnect(done);
         });
     });
 
+    describe('Global sequences => ', function () {
 
+        describe('a simple id field => ', function () {
```

Comment:
Can you add a test case for simple, non-exclusive counters?

Reply:
ok